packages/chart/src/Chart.tsx

```diff
@@ -506,7 +508,11 @@ class Chart extends Component<ChartProps, ChartState> {
         });
 
         const { onError } = this.props;
-        onError(new DownsamplingError(downsamplingError));
+        onError(
+          new DownsamplingError(
+            downsamplingError == null ? undefined : `${downsamplingError}`
```

Comment:
This is technically a runtime change, but the DownsamplingError constructor should be stringifying it's first arg anyway.